mrmustard/lab/gates.py

```diff
@@ -797,3 +801,69 @@ def __init__(
     @property
     def Y_matrix(self):
         return gaussian.noise_Y(self.noise.value, settings.HBAR)
+
+
+class PhaseNoise(Parametrized, Transformation):
+    r"""The phase noise channel.
+
+    The phase noise channel is a non-Gaussian transformation that is equivalent to
+    a random phase rotation.
+
+    Args:
+        phase_stdev (float or List[float]): the standard deviation of the (wrapped) normal
+            distribution for the angle of the rotation
+        modes (optional, list(int)): the single mode this gate is applied to (default [0])
```

Comment:
`modes` (with an s) is a list of int but the description states **single**, a bit confusing